Detections/AzureDevOps/AzDOHistoricPrPolicyBypassing.yaml

```diff
@@ -19,35 +19,19 @@ relevantTechniques:
   - T1098
```

Comment:
there are 3 Tactics but only one relevantTechniques. I think we can keep only Persistence, unlesss we add other techniques matching to Tactics

Reply:
done